### PR TITLE
Increase visibility of CAIRR pipeline in the submission manual

### PR DESCRIPTION
**Pull request by marcosmro**
As we agreed on the MiniStd WG Call on 2019-07-12, I have updated the submission pipeline to give more visibility to the CAIRR pipeline. I have described the steps to complete a submission to the NCBI repositories using the CAIRR pipeline and included some screenshots of the CEDAR UI.

@ahmadchan has already reviewed these updates and suggested some minor edits that I have already applied (see https://github.com/ahmadchan/airr-standards/commit/eb66901c3b31cc67ae4fd67d3a93bcfe04fec6f8).

### CODE REVIEW

**Comment by marcosmro** [comment]
I've updated the CAIRR pipeline page (https://docs.airr-community.org/en/latest/cairr/overview.html) to point to the CAIRR pipeline section in the submission manual and to remove all the information and screenshots that were out of date.

**Comment by scharch** [comment]
I think the CAIRR pipeline page needs to remind users explicitly that there
are other steps afterward needed for MiAIRR compliance.

**Comment by marcosmro** [comment]
I don't know what those steps are. @scharch or @ahmadchan do you want to take a stab at extending the page to clarify that?

**Comment by scharch** [comment]
I can take a crack at it Monday or Tuesday

**Comment by marcosmro** [comment]
Great. Thanks!

File: docs/miairr/manual_miairr_ncbi.rst [hunk 1]
```diff
@@ -5,126 +5,131 @@ MiAIRR-to-NCBI Submission Manual
 Scope of this document
 ======================
 
-Provide a user manual describing the submission of AIRR data using
-the NCBI reference implementation described in [Rubelt_2017]_.
-This implementation uses NCBI’s BioProject, BioSample, Sequence Read
-Archive (SRA) and GenBank repositories and metadata standards to report
-AIRR data.
+Provide a user manual describing the submission of AIRR data using the NCBI reference implementation described in `[Rubelt_2017] <https://doi.org/10.1038/ni.3873>`_. This implementation uses NCBI’s BioProject, BioSample, Sequence Read Archive (SRA) and GenBank repositories and metadata standards to report AIRR data.
```
Comment (bussec) [inline docs/miairr/manual_miairr_ncbi.rst]: Please be consistent with the type of apostrophes used within a single document ( ' vs. ’ ).

File: docs/cairr/overview.rst [hunk 2]
```diff
@@ -1,74 +1,30 @@
...
...udies  at the NCBI are often described using inconsistent terminologies, limiting scientists’ ability to access, find, interoperate, and reuse the data sets and to understand how the experiments were performed. CEDAR (Center for Expanded Data Annotation and Retrieval) develops technologies involving the use of data standards and ontologies to improve metadata quality. In order to improve metadata quality and ease AIRR-seq study submission process, we have developed an AIRR-seq data submission pipeline named CEDAR-AIRR (CAIRR). CAIRR leverages CEDAR’s technologies to:  i) create web-based templates whose entries are controlled by ontology terms, ii) generate and validate metadata and iii) submit the ontology-linked metadata and sequence files (FASTQ) to the NCBI BioProject, BioSample, and Sequence Read Archive (SRA) databases. Thus, CAIRR provides a web-based metadata submission interface that supports compliance with MiAIRR standards. The interface enables ontology-based validation for several data elements, including: organism, disease, cell type and subtype, and tissue. This pipeline will facilitate the NCBI submission process and improve the metadata quality of AIRR-seq studies.
```
Comment (bussec) [inline docs/cairr/overview.rst]: Please be nice to Git ;-) Therefore use one line for each sentence, this simplifies review with `diff`-like tools.

File: docs/cairr/overview.rst [hunk 3]
```diff
@@ -1,74 +1,30 @@
...
-Create your metadata. Go to http://cairr.miairr.org, and CEDAR should open in your browser. (If you are not already logged in, you may need to log in before being redirected to the metadata page.) It will look something like this. 
-
-
-   .. image:: ./images/miairr_cedartemplate.png
-
+Submission Steps
+----------------
 
-If you do not want your metadata to be public immediately in NCBI, fill out the Submissions Release Data field at the top of the form. Then click on any of the three metadata sections to open them up.
+The submission steps are described in the `MiAIRR-to-NCBI Submission Manual (Option 1. Submission via the CEDAR system) <https://github.com/airr-community/airr-standards/blob/master/docs/miairr/manual_miairr_ncbi.rst#option-1-submission-via-the-cedar-system-cairr-submission-pipeline>`_. You will need a CEDAR system account; you can self-register at  https://cedar.metadatacenter.org.  You will also need the identifier of a BioProject already entered in the NCBI BioProject database.
```
[the remark this answers is not in the document]
Comment (marcosmro) [reply docs/cairr/overview.rst]: Done.

File: docs/cairr/overview.rst [hunk 4]
```diff
@@ -1,74 +1,30 @@
...
+----------------
 
 Please let us know how it went!  If you are willing, we’d love to have your comments in a `short survey <https://www.surveymonkey.com/r/your-metadata-experience>`_, it should just take 5 minutes or so. 
 
 We also welcome entry of issues and requests in our `github repository issues <https://github.com/metadatacenter/cedar-project/issues>`_, and emails can be sent to cedar-users@lists.stanford.edu. Both of these resources are publicly visible. 
+
+Support or Contact
+------------------
+
+Having trouble with NCBI submission process through our pipeline? Please email to `Syed Ahmad Chan Bukhari <mailto:ahmad.chan@yale.edu>`_ or to `Marcos Martínez-Romero <mailto:marcosmr@stanford.edu>`_ and we’ll help you sort it out.
```
[the remark this answers is not in the document]
Comment (marcosmro) [reply docs/cairr/overview.rst]: Done.

File: docs/cairr/overview.rst (hunk 2 above)
Comment (marcosmro) [reply docs/cairr/overview.rst]: Done. Please, take a look to see if it looks good now.

File: docs/cairr/overview.rst [hunk 5]
```diff
@@ -1,74 +1,30 @@
...
-**Cite MiAIRR Pipeline**
-
-Bukhari, Syed Ahmad Chan, Martin J. O'Connor, Marcos Martínez-Romero, Attila L. Egyedi, Debra Debra Willrett, John Graybeal, Mark A. Musen, Florian Rubelt, Kei H. Cheung, and Steven H. Kleinstein. "The CAIRR pipeline for submitting standards-compliant B and T cell receptor repertoire sequencing studies to the NCBI." Frontiers in Immunology 9 (2018): 1877. DOI: 10.3389/fimmu.2018.01877 (now in press)
-
-
-**Tell Us About It**
+Tell Us About It
+----------------
 
 Please let us know how it went!  If you are willing, we’d love to have your comments in a `short survey <https://www.surveymonkey.com/r/your-metadata-experience>`_, it should just take 5 minutes or so.
```
[the remark this answers is not in the document]
Comment (marcosmro) [reply docs/cairr/overview.rst]: Done.

**Comment by marcosmro** [comment]
Thanks for your feedback, @bussec. I have applied all your requested changes.

File: docs/miairr/manual_miairr_ncbi.rst (hunk 1 above)
Comment (marcosmro) [reply docs/miairr/manual_miairr_ncbi.rst]: Done.

**Comment by scharch** [comment]
It wasn't this file I wanted to edit, but I made some proposed changes in a pull request I just opened on your branch @marcosmro --didn't pay attention to where I was forking from, can re-add those edits elsewhere if that is more appropriate.

**Comment by marcosmro** [comment]
I've merged the changes requested by @scharch into my branch. These changes have been added to the pull request.

**Pull request merged**